postgres/postgres.go

```diff
@@ -0,0 +1,33 @@
+package postgres
```

Comment:
Rename file/folder to database instead of postgres, as we may use a different database (cloud db in our case) later.